src/rdiff_backup/Globals.py

```diff
@@ -20,10 +20,14 @@
 
 import re
 import os
-from . import Version
 
 # The current version of rdiff-backup
-version = Version.version
+# Get it from package info or fall back to DEV version.
+try:
+    import pkg_resources
+    version = pkg_resources.get_distribution("rdiffweb").version
+except:
+    version = "DEV"
```

Comment:
I get DEV when I run `PYTHONPATH=$PWD/build/lib.linux-x86_64-3.7 PATH=$PWD/build/scripts-3.7:$PATH rdiff-backup --version` (see DEVELOP.md). Would there be a way to get the calculated version without installing?

I'm wondering if we should put a more explicit ("Not.Installed.See.Readme.Md"), or a tracked ("1.3.6") version, or a combination of both ("1.3.6.SeeREADMEmd"), also because this version will appear if someone forgets to install setuptools.

Reply:
Another thought which might make everybody happy: replace the line 30 with the following, obviously keeping Version.py in this case
```python
except:
    from . import Version
    version = Version.version
```
This means that we'd need to track the version as Git tag _and_ in Version.py, but that would be the price to pay to make everybody happy. I would put the first as v1.3.6 and the 2nd as 1.3.6.0 so that we can differentiate and the generated version will always be higher unless it is exactly the same (Git) version.

Reply:
If we go with scm_version, the goal is to avoid tracking the version manually. Keeping it in the code doesn't seem a very nice solution.

> I get DEV when I run PYTHONPATH=$PWD/build/lib.linux-x86_64-3.7 PATH=$PWD/build/scripts-3.7:$PATH rdiff-backup --version (see DEVELOP.md). Would there be a way to get the calculated version without installing?

You need to run either "install" or "develop" on the project to get the version. 

We could get the version without installing if I add "setuptools_scm" as a dependencies.

Reply:
`./setup.py develop --user` is almost good, it leaves only little traces on my system. And I see that `./setup.py sdist` creates a source package that @ottok could use to do his Debian package and which doesn't forget the version generated when taken out of Git. If Otto is fine with it, he can resolve the conversation.

Reply:
I would still prefer a more explicit "NotInstalledDevVersion" or similar default version.